src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java

```diff
@@ -45,22 +45,26 @@
  */
 public class TestSetupHooks {
 
-  @Before(order = 2, value = "@BQ_SINK_TEST")
-  public void setTempTargetBQTable() {
-    String bqTargetTable = "TestSN_table" + RandomStringUtils.randomAlphanumeric(10);
-    PluginPropertyUtils.addPluginProp("bqTargetTable", bqTargetTable);
-    BeforeActions.scenario.write("BigQuery Target table name: " + bqTargetTable);
+  @Before(order = 1, value = "@CREATE_CUSTOM")
+  public static void createCustom() throws UnsupportedEncodingException, ParseException {
```

Comment:
Instead of writing @CREATE_CUSTOM use something @CREATE_TEST_DATA and @DELETE_TEST_DATA

Reply:
done